lib/fluent/log.rb

```diff
@@ -292,7 +292,11 @@ def event(level, args)
       }
 
       map.each_pair {|k,v|
-        message << " #{k}=#{v.inspect}"
+        if k == "error".freeze
+          message << " error_class=#{v.class.to_s} error=#{v.to_s.inspect}"
```

Comment:
Need to add another condition `!map.has_key?('error_class')`, otherwirse, ` error_class=` appears twice for old codes such as:

```
log.error 'message', error_class: e.class, error: e
```